clusters/hive/pools/ocp-release-4.9.4-x86-64-for-4.8.0-0-to-4.9.0-0_clusterimageset.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: hive.openshift.io/v1
+kind: ClusterImageSet
+metadata:
+  name: ocp-4.8.latest-amd64
+  annotations:
+    version_lower: 4.8.0-0
+    version_upper: 4.9.0-0
```

Comment:
I think you just need to label the pool, and then the bumper makes sure it refers to an automatically bumped clsuterimageset.

Reply:
Updated to only change the labels, PTAL